lib/scripts/sso.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+# shellcheck disable=SC2016 disable=SC1117
+
+set -euo pipefail; [[ -z ${TRACE:-}  ]] || set -x
+
+export DEBIAN_FRONTEND=noninteractive
+
+if [[ ! -f /etc/apt/sources.list.d/lemonldap-ng.list ]]; then
+	cat >/etc/apt/sources.list.d/lemonldap-ng.list <<-EOF
+		deb     https://lemonldap-ng.org/deb 2.0 main
+	EOF
+	curl -fsSL https://lemonldap-ng.org/_media/rpm-gpg-key-ow2 | apt-key add -
+
+	apt-get -y update
+fi
+
+ufw disable
+
+apt-get -y install lemonldap-ng \
+	lemonldap-ng-fastcgi-server \
+	lemonldap-ng-doc \
+	perlmagick \
+	liblasso-perl \
+	libio-string-perl \
+	libxml-libxslt-perl \
+	libxml-simple-perl \
+	libunicode-string-perl \
+	libsoap-lite-perl \
+	libregexp-common-perl \
+	libnet-ldap-perl \
+	libmime-tools-perl \
+	libgd-securityimage-perl \
+	libemail-sender-perl \
+	libdigest-hmac-perl \
+	libdbi-perl \
+	libredis-perl \
+	libstring-random-perl \
+	libapache-session-browseable-perl \
+	libcgi-emulate-psgi-perl \
+	libcgi-compile-perl \
+	cpanminus
+
+sed -i 's/example\.com/sso\.vagrant\.ga/g' /etc/lemonldap-ng/* /var/lib/lemonldap-ng/conf/lmConf-1.json
+sed -i -e 's/^;checkTime.*/checkTime = 1/' /etc/lemonldap-ng/lemonldap-ng.ini
```

Comment:
```suggestion
sed -i 's/^;checkTime.*/checkTime = 1/' /etc/lemonldap-ng/lemonldap-ng.ini
```

P.S. Sed'de `-e` anahtarını birden fazla expresion girebilmek için kullanıyoruz, tek expressionlarda kullanımı gereksiz (benim de sıkça yaptığım bir şey).